tests/Microsoft.Bot.Builder.Azure.Tests/BlobsTranscriptStoreTests.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using Microsoft.Bot.Builder.Azure.Blobs;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+// These tests require Azure Storage Emulator v5.7
+// The emulator must be installed at this path C:\Program Files (x86)\Microsoft SDKs\Azure\Storage Emulator\AzureStorageEmulator.exe
+// More info: https://docs.microsoft.com/azure/storage/common/storage-use-emulator
```

Comment:
Is there a performance issue by using Storage Emulator with unit tests?

Reply:
Yes, there is definitely a performance issue with the storage emulator.  Some have mentioned we should maybe put together a daily run of storage integration tests, rather than unit tests against the storage emulator.